build.gradle

```diff
@@ -141,6 +141,8 @@ dependencies {
 		exclude module: 'guava'
 		exclude module: 'gson'
 	}
+
+	compile group: 'com.carrotsearch', name: 'hppc', version: '0.7.1'

```

Comment:
...why import a new dependency when a version (`it.unimi.dsi:fastutil:7.0.12_mojang`) of [fastutil](http://fastutil.di.unimi.it/) is included with Minecraft already?
cc @Barteks2x